library/ssl_tls.c

```diff
@@ -735,21 +619,12 @@ static int tls_prf_sha384( const unsigned char *secret, size_t slen,
 
 static void ssl_update_checksum_start( mbedtls_ssl_context *, const unsigned char *, size_t );
 
-#if defined(MBEDTLS_SSL_PROTO_SSL3) || defined(MBEDTLS_SSL_PROTO_TLS1) || \
-    defined(MBEDTLS_SSL_PROTO_TLS1_1)
-static void ssl_update_checksum_md5sha1( mbedtls_ssl_context *, const unsigned char *, size_t );
-#endif
-
 #if defined(MBEDTLS_SSL_PROTO_SSL3)
```

Comment:
As per @mpg comment above, this block should be removed (also in other places - these should all go away once rebased on top of the latest development)

Reply:
OK